Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -99,26 +136,50 @@ open class PerformanceTestCase: XCTestCase {
         }
         return measures.average
     }
+
+    public func measureMutable(
```

Comment:
That is to say, `closure` in one place is a `() -> Double`, while it is `() -> Void` in another. I'm not sure (just looking at it immediately — though perhaps that's not an unreasonable metric) exactly what the nature of the measurement is.